packages/design/src/animation.mdx

```diff
@@ -0,0 +1,47 @@
+---
+name: Animation
+menu: Design
+route: /animation
+---
+
+# Animation
+### Patterns
+
+Animations in Jobber fall under categories based on their purpose:
+
+**Reveal** 
+
+Reveals are used to show and hide information on the screen. For example, toasts, modals, sidebars appearing and disappearing on the screen.
+
+**State change**
+
+State changes are used to inform the user that a context change is taking place. This is used when the object has changed state due to user interaction (e.g. hover and focus states).
+
+**Emphasis**
+
+Emphasis is used to bring attention to something on already on the screen. For example, when you want to guide users to a particular part of the interface for onboarding or learning purposes.
+
+**Delighter**
+
+Delighters can be used occasionally to create surprise and delight, and can give personality to our brand. These can be used to reinforce success in an important milestones in a user's journey.
+
+### Timing
+Timing is used for animation durations.
+
+Use judgement when determining the appropriate timing. Duration will depend on the size of the element and the distance it covers. Animations for larger elements or longer distances should be longer than smaller elements or shorter distances.
+
+
+| Name        | Value                  | Examples of use              |
+| :--------------- | :--------------------- | :--------------------- |
+| `--duration-fastest` | 100ms    | smaller elements exiting, small elements, state changes
```

Comment:
The way these are written, are these going to be animation "tokens" that can be used in React? Or are they CSS variables?

Reply:
I was thinking they would be like tokens, similar to what we are doing currently for spacing